app/src/main/java/com/aurora/carevision/core/component/CVTopAppBar.kt

```diff
@@ -0,0 +1,152 @@
+package com.aurora.carevision.core.component
+
+import androidx.compose.foundation.Image
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.aurora.carevision.R
+import com.aurora.carevision.app.ui.theme.CVTheme
+import com.aurora.carevision.app.ui.theme.White
+
+@Composable
+fun TopAppBarBackLeft(
+    title: String,
```

Comment:
title text가 비어있고 뒤로가기 버튼만 있는 탑앱바도 있을 수 있으니 이렇게 default값을 넣어주면 좋을 것 같아요
```suggestion
    title: String = "",
```